pkg/sdn/plugin/proxy.go

```diff
@@ -78,7 +79,7 @@ func (proxy *ovsProxyPlugin) watchEgressNetworkPolicies() {
 			policy.Spec.Egress = nil
 		}
 		proxy.updateNetworkPolicy(*policy)
-		// FIXME: poke the endpoint-syncer somehow...
+		proxy.OnEndpointsUpdate(proxy.lastEndpoints)
```

Comment:
what happens if this gets called before `proxy.lastEndpoints` is set for the first time? will sending an empty list to `OnEndpointsUpdate` do anything bad?